scripts/test-wasm.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+cargo test -F test-js
```

Comment:
This doesn't appear to consume `WASM_PACK_BUILD_FLAGS`, or is there some magicks deep in the build chain that does so?  (That would, I guess, explain why that env var is being used in the first place...)  :shrug: